neovim/msgpack_rpc/event_loop/asyncio.py

```diff
@@ -111,6 +114,11 @@ def _threadsafe_call(self, fn):
         self._loop.call_soon_threadsafe(fn)
 
     def _setup_signals(self, signals):
+        if os.name == 'nt':
+            # add_signal_handler is not supported in win32
+            self._signals = []
+            return
```

Comment:
Currently signal support is not working - https://github.com/neovim/python-client/issues/157#issuecomment-183994226 and https://github.com/neovim/python-client/issues/157#issuecomment-183997724 - maybe add a TODO: here?